backend/src/routes/userRoute.ts

```diff
@@ -0,0 +1,66 @@
+import express, { Request, Response } from "express";
+import admin, { ServiceAccount } from "firebase-admin";
+
+import * as serviceAccount from "../firebase/ServiceAccountKey.json";
+import User, { UserDocument } from "../models/User";
+
+// Define the type for req.body
+type CreateUserRequestBody = {
+  name: string;
+  gender: string;
+  accountType: "admin" | "team";
+  approvalStatus: string; // NOTE Should this be restricted to certain values?
+  email: string;
+  password: string;
+};
+
+const router = express.Router();
+
+// Initialize Firebase Admin SDK
+admin.initializeApp({
```

Comment:
Lets initialize the firebase app in a different area and export the app so that its accessible to other files. See [this](https://github.com/TritonSE/ALUM-Mobile-Application/blob/main/backend/src/services/firebase.ts) as an example